site/components/BlockContentRenderer/index.tsx

```diff
@@ -82,6 +82,14 @@ const serializers: BlockContentProps["serializers"] = {
         </A>
       );
     },
+    internalLink: ({ children, mark }) => {
+      const { href } = mark;
+      return (
+        <A className={styles.link} href={`${href}?dl=`}>
```

Comment:
lets stick with browser default i dont think most people want to save it

```suggestion
        <A className={styles.link} href={`${href}`}>
```

Reply:
good call that would come off as fishy